rocAL/rocAL_pybind/examples/rocAL_api_coco_pipeline.py

```diff
@@ -58,170 +57,171 @@ def __init__(self, pipelines, tensor_layout=types.NCHW, reverse_channels=False,
+        brightness = fn.uniform(rng_range=[0.875, 1.125])
+        hue = fn.uniform(rng_range=[5.0, 170.0])
+        ct_images = fn.color_twist(
+            res_images, saturation=saturation, contrast=contrast, brightness=brightness, hue=hue)
         flip_coin = fn.random.coin_flip(probability=0.5)
-        images = fn.resize_mirror_normalize(images_decoded, device="gpu",
-                                            image_type=types.RGB,
-                                            resize_width=crop, resize_height=crop,
-                                            mirror=flip_coin,
-                                            output_layout = types.NCHW,
-                                            output_dtype = types.FLOAT,
-                                            mean=[0,0,0],
-                                            std=[1,1,1])
-        matched_idxs  = fn.box_iou_matcher(anchors=anchor_list, criteria=0.5,
-                                     high_threshold=0.5, low_threshold=0.4,
-                                     allow_low_quality_matches=True)
-        coco_train_pipeline.set_outputs(images)
-    coco_train_pipeline.build()
-    COCOIteratorPipeline = ROCALCOCOIterator(coco_train_pipeline)
-    cnt = 0
-    for epoch in range(1):
-        print("+++++++++++++++++++++++++++++EPOCH+++++++++++++++++++++++++++++++++++++",epoch)
-        for i, it in enumerate(COCOIteratorPipeline):
-            print("************************************** i *************************************",i)
-            print(it)
-            for i, img in enumerate(it[0]):
-                draw_patches(img, cnt, it[1]['boxes'][i], "cpu")
-                cnt =  cnt + 1
-        COCOIteratorPipeline.reset()
-    print("*********************************************************************")
+        cmn_images = fn.crop_mirror_normalize(ct_images,
+                                              crop=(224, 224),
+                                              crop_pos_x=0.0,
+                                              crop_pos_y=0.0,
+                                              mean=[0, 0, 0],
+                                              std=[1, 1, 1],
+                                              mirror=flip_coin,
+                                              output_layout=tensor_format,
+                                              output_dtype=tensor_dtype)
+        pipe.set_outputs(cmn_images)
+    # Build the pipeline
+    pipe.build()
+    # Dataloader
+    if (args.rocal_gpu):
+        data_loader = ROCALCOCOIterator(
+            pipe, multiplier=pipe._multiplier, offset=pipe._offset, display=display, tensor_layout=tensor_format, tensor_dtype=tensor_dtype, device="cuda")
+
+    else:
+        data_loader = ROCALCOCOIterator(
+            pipe, multiplier=pipe._multiplier, offset=pipe._offset, display=display, tensor_layout=tensor_format, tensor_dtype=tensor_dtype, device="cpu")
+
+    import timeit
```

Comment:
All imports generally will go in starting part of code
Please move it to the top